Samples.Objc/Samples.ObjC/AppDelegate.m

```diff
@@ -1,37 +1,45 @@
 //
 //  AppDelegate.m
-//  PayPalCheckout-Samples-iOS-Objc
+//  PayPalNativeCheckoutObjC
 //
 //  Created by Haider Khan on 5/23/20.
 //  Copyright © 2020 PayPal. All rights reserved.
 //
 
 #import "AppDelegate.h"
+#import "PayPalAPI.h"
+@import PayPalCheckout;
 
 @interface AppDelegate ()
-
 @end
 
 @implementation AppDelegate
 
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-  // Override point for customization after application launch.
+  PPCheckoutConfig *config = [[PPCheckoutConfig alloc] initWithClientID:[PayPalAPI.shared clientId]
+                                                              returnUrl:@"<return_url>"
+                                                            createOrder:nil
+                                                              onApprove:nil
+                                                               onCancel:nil
+                                                                onError:nil
+                                                            environment:PPCEnvironmentSandbox];
+  [PPCheckout setConfig:config];
   return YES;
 }
 
 
 #pragma mark - UISceneSession lifecycle
 
 
-- (UISceneConfiguration *)application:(UIApplication *)application configurationForConnectingSceneSession:(UISceneSession *)connectingSceneSession options:(UISceneConnectionOptions *)options API_AVAILABLE(ios(13)) {
+- (UISceneConfiguration *)application:(UIApplication *)application configurationForConnectingSceneSession:(UISceneSession *)connectingSceneSession options:(UISceneConnectionOptions *)options {
   // Called when a new scene session is being created.
   // Use this method to select a configuration to create the new scene with.
   return [[UISceneConfiguration alloc] initWithName:@"Default Configuration" sessionRole:connectingSceneSession.role];
 }
 
 
-- (void)application:(UIApplication *)application didDiscardSceneSessions:(NSSet<UISceneSession *> *)sceneSessions API_AVAILABLE(ios(13)) {
+- (void)application:(UIApplication *)application didDiscardSceneSessions:(NSSet<UISceneSession *> *)sceneSessions {
   // Called when the user discards a scene session.
```

Comment:
Please revert back the changes here for `API_AVAILABLE(ios(13))` our deployment target is `iOS 11.0`. We will see the following warnings with this deletion:

![image](https://user-images.githubusercontent.com/7844550/112887594-38555300-9088-11eb-8155-c58546123573.png)